ProgramFiles/Utilities/curvature_mode_toolbox/illustrate_backbone.m

```diff
@@ -0,0 +1,64 @@
+function illustrate_backbone(curvdef,paramvalue,orientation,plotnum)
```

Comment:
Do you know why this is showing up as entirely new and seems to cut off in the middle of the file?